en/components/lineargauge.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Linear Graph Component - Native Angular | Ignite UI for Angular 
+_description: In Ignite UI for Angular, use the Linear Gauge component to see a simple display of a value compared against a scale and one or more ranges.  
+_keywords: Ignite UI for Angular, Angular, Native Angular Components Suite, Native Angular Controls, Native Angular Components, Native Angular Components Library, Angular Grid, Angular Data Grid, Angular Grid Control, Angular Grid Component, Angular linear graph Component, Angular linear graph 
+---
+## Linear Gauge
+
+In Ignite UI for Angular, use the Linear Gauge component to see a simple display of a value compared against a scale and one or more ranges.
+
+### Demo
+
+<div class="sample-container" style="height: 550px">
+    <iframe id="linear-gauge-sample-iframe" src='{environment:demosBaseUrl}/linear-gauge-sample' width="100%" height="100%" seamless frameBorder="0" onload="onSampleIframeContentLoaded(this);"></iframe>
+</div>
+<div>
+    <button data-localize="stackblitz" class="stackblitz-btn"   data-iframe-id="linear-gauge-sample-iframe" data-demos-base-url="{environment:demosBaseUrl}">View on StackBlitz
+    </button>
+</div>
+
+<div class="divider--half"></div>
+
+The features of the linear gauge control include configurable orientation and direction, configurable visual elements, and more. The control has also a built-in support for animated transitions as well as tooltips.
+
+### Dependencies
+The linear gauge exported as an `NgModule`, you need to import the _IgxLinearGaugeModule_ inside your `AppModule`:
+
+```typescript
+// app.module.ts
+import { IgxLinearGaugeModule } from '@infragistics/ignite-ui-xplatform-for-angular/Debug/ES5/ig-lineargauge-module';
+
+@NgModule({
+    imports: [
+        ...
+        IgxLinearGaugeModule,
+        ...
+    ]
+})
+export class AppModule {}
+```
+
+<div class="divider--half"></div>
+
+### Usage
+
+The following code demonstrates how create a linear gauge containing a needle and three comparative ranges on the scale.
+
+```html
+ <igx-linear-gauge width="70px"
+                   height="300px"
+                   minimumValue = "5"
+                   maximumValue = "55"
+                   value = "43">
+    <igx-linear-gauge.ranges>
+        <igx-linear-graph-range startValue="0"
+                                endValue="15"
+                                brush="red"/>
+        <igx-linear-graph-range startValue="15"
+                                endValue="30"
+                                brush="yellow"/>
+        <igx-linear-graph-range startValue="30"
+                                endValue="55"
+                                brush="green"/>
+    </igx-linear-gauge.ranges>         
```

Comment:
please remove `</igx-linear-gauge.ranges>` closing tag because it will throw errors! It is needed only in XAML